python/paddle/fluid/tests/unittests/transformer_model.py

```diff
@@ -403,7 +403,7 @@ def transformer(
         trg_pad_idx,
         pos_pad_idx, ):
     file_obj = fluid.layers.open_recordio_file(
-        filename='./wmt16.recordio',
+        filename='./wmt16_test_pe.recordio',
```

Comment:
why is this changed?

Reply:
can we set this to /tmp/wmt16.recordio? I have a change for distributed training test and want to share this transformer_model